4.0.0-alpha.2/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM jetty:9-jre8
+
+ENV GN_FILE geonetwork.war
+ENV GN_VERSION 4.0.0-alpha.2
+ENV GN_DOWNLOAD_MD5 TODO
+
+USER root
+RUN apt-get -y update && \
+    apt-get -y install vim \
+    wget \
+    curl \
+    sudo
+
+RUN mkdir -p /catalogue-data && \
+    chown -R jetty:jetty /catalogue-data
+
+RUN mkdir -p /var/lib/jetty/webapps/geonetwork && \
+    chown -R jetty:jetty /var/lib/jetty/webapps/geonetwork
+
+USER jetty
+
+RUN cd /var/lib/jetty/webapps/geonetwork/ && \
+    curl -fSL -o $GN_FILE \
+     https://sourceforge.net/projects/geonetwork/files/GeoNetwork_unstable_development_versions/${GN_VERSION}/geonetwork.war/download && \
+     echo "$GN_DOWNLOAD_MD5 *${GN_FILE}" | md5sum -c \
+     unzip geonetwork.war && \
+     rm geonetwork.war
+
+COPY ./docker-entrypoint.sh /entrypoint.sh
+ENTRYPOINT ["/entrypoint.sh", "jetty.start"]
```

Comment:
I can't execute something like `docker run -it geonetwork:4.0.0-alpha2 bash` to start the container with a bash session open. Instead it starts jetty because the first argument of `entrypoint.sh` is always `jetty.start`.

I would add a `CMD ["jetty.start"]` and remove it from the `ENTRYPOINT` instruction:
```Dockerfile
ENTRYPOINT ["/entrypoint.sh"]
CMD [ "jetty.start" ]
```